pkg/cmd/server/api/v1/types.go

```diff
@@ -76,6 +76,9 @@ const (
 type MasterConfig struct {
 	v1.TypeMeta `json:",inline"`
 
+	// If Openshift is enabled
+	OpenshiftEnabled bool `json:"openshiftEnabled"`
+
```

Comment:
What about all the existing server configurations that are missing this value (which will set the bool field to false on deserialization, and disable openshift features)? We have to preserve existing behavior for those configurations on update